assets/javascript/game.js

```diff
@@ -0,0 +1,60 @@
+// Array of computer choices and user choices
+var computerChoices = ["a", "b", "c", "d", "e", "f", "g", "h", "i", "j", "k", "l", "m"
+, "n", "o", "p", "q", "r", "s", "t", "u", "v", "w", "x", "y", "z"]
+
+
+//Wins, loses and guesses starting point
+    var wins = 0;
+    var losses = 0;
+    var guessesLeft = 9;
+    var guessesArray = [];
+
+console.log(guessesLeft)
+console.log(guesses)
+
+
+// Display variables
+var winsText = document.getElementById("wins-text");
+var lossesText = document.getElementById("losses-text");
+var guessesLeftText = document.getElementById("guessesLeft-text");
+var guessesArrayText = document.getElementById("guessesArray-text");
+
+//display
+winsText.textContent = "wins: " + wins;
+lossesText.textContent = "losses: " + losses;
+guessesLeftText.textContent = "Guesses Left: " + guessesLeft;
+guessesArrayText.textContent = "Your Guesses so far: " + guessesArray;
+ 
+// Randomly chooses a choice from the alphabet array
+var computerGuess = computerChoices[Math.floor(Math.random() * computerChoices.length)];
+console.log(computerGuess)
+
+    // User choice determined by pressing a key, function operates after key is pressed
+ document.onkeyup = function(event) {
+    var userGuess = event.key;
+console.log(userGuess)
```

Comment:
you should try to add a condition here to only accept alphabet keys and not any other keys